### PR TITLE
[GRW-670] adding new icons for the homepage

### PR DESCRIPTION
**Pull request by ironprice91**
Adding new icons via `yarn icons`

### CODE REVIEW

**Comment by igor-p** [comment]
When I compile/pack and check this out in the `example` directory, I see that what should be the outlined table shows up as filled:
image.

The filled one looks fine. I think the outlined one just needs `fill: "none"`. It looks like all icons have `fill: "currentColor"` by default, and it seems like the icons in the figma don't have that.

**Review by alexchan7432 — DISMISSED** [review]

File: src/icons/table-outline.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import SvgIcon, { SvgIconProps } from '@material-ui/core/SvgIcon';
+
+export function IconTableOutline(props: SvgIconProps) {
+  return (
+    <SvgIcon viewBox="0 0 24 24" {...props}>
+      <path d="M20 3H5C3.9 3 3 3.9 3 5V8H22V5C22 3.9 21.1 3 20 3Z" stroke="black" />
+      <path d="M20 21H17V10H22V19C22 20.1 21.1 21 20 21Z" stroke="black" />
+      <path d="M15 10.02H10V21H15V10.02Z" stroke="black" />
+      <path d="M5 21C3.9 21 3 20.1 3 19V10H8V21H5Z" stroke="black" />
```
Comment (herrethan) [inline src/icons/table-outline.tsx:10]: This looks wrong. Check the figma file and with design. No icons should have a black stroke. 
Likely the it was not drawn using the previous conventions.

**Comment by ironprice91** [comment]
@igor-p @herrethan Thanks for catching that. Alee updated the icon file and the `table-outline` icon should be showing up correctly now. I also updated the `parseSvgContents` to stip out and `stroke` attributes


https://github.com/reonomy/styles/pull/67/commits/1104831ddf7649d8d3480f9ee1aab4ae0884b59d
Screen Shot 2021-09-27 at 4 47 25 PM

File: scripts/utils.js [hunk 2]
```diff
@@ -27,7 +27,8 @@ function parseSvgContents(string) {
     .replace(/\n/g, '') // strip line breaks
     .replace(/<svg [^>]+>/, '') // strip opening svg tag
     .replace(/<\/svg>/, '') // strip ending svg tag
-    .replace(/ fill=".+?"/g, ''); // strip any fill="" color definitions
+    .replace(/ fill=".+?"/g, '') // strip any fill="" color definitions
+    .replace(/ stroke=".+?"/g, ''); // strip any stroke="" color definitions
```
Comment (herrethan) [inline scripts/utils.js:31]: 👍

File: src/icons/grab-filled.tsx [hunk 3]
```diff
@@ -6,7 +6,6 @@ export function IconGrabFilled(props: SvgIconProps) {
     <SvgIcon viewBox="0 0 24 24" {...props}>
       <path
         d="M21.25 12C21.25 17.1086 17.1086 21.25 12 21.25C6.89137 21.25 2.75 17.1086 2.75 12C2.75 6.89137 6.89137 2.75 12 2.75C17.1086 2.75 21.25 6.89137 21.25 12ZM8.5 9C8.5 8.17157 7.82843 7.5 7 7.5C6.17157 7.5 5.5 8.17157 5.5 9V15C5.5 15.8284 6.17157 16.5 7 16.5C7.82843 16.5 8.5 15.8284 8.5 15V9ZM13.5 9C13.5 8.17157 12.8284 7.5 12 7.5C11.1716 7.5 10.5 8.17157 10.5 9V15C10.5 15.8284 11.1716 16.5 12 16.5C12.8284 16.5 13.5 15.8284 13.5 15V9ZM18.5 9C18.5 8.17157 17.8284 7.5 17 7.5C16.1716 7.5 15.5 8.17157 15.5 9V15C15.5 15.8284 16.1716 16.5 17 16.5C17.8284 16.5 18.5 15.8284 18.5 15V9Z"
-        stroke="#1C1D20"
```
Comment (herrethan) [inline src/icons/grab-filled.tsx:9]: ...🤔

**Review by herrethan — APPROVED** [review]
Looks good! 
May want to double check the `IconGrabFilled` once you promote to the app and make sure the grabby styles are still copacetic.

**Pull request merged**